agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -120,6 +120,16 @@ bool backhaul_manager::init()
         return false;
     }
 
+    if (m_sConfig.ucc_listener_port != 0) {
+        m_agent_ucc_listener = std::make_unique<agent_ucc_listener>(
+            *this, m_sConfig.ucc_listener_port, m_sConfig.vendor, m_sConfig.model,
+            m_sConfig.bridge_iface);
+        if (m_agent_ucc_listener && !m_agent_ucc_listener->start("ucc_listener")) {
```

Comment:
perhaps the name should be `prplmesh_agent_ucc_listener`?

Reply:
It is for logging purposes, so I think it is better to use the shortest name possible. Some places leave it empty...